style.css

```diff
@@ -0,0 +1,358 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body {
+  background-image: url('./images/header-shapes-mobile.svg');
+  background-repeat: no-repeat;
+  background-position: 0 5rem;
+  background-color: #f7f7f9;
+}
+
+.main_heading {
+  color: blue;
+  padding-left: 30%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  width: 100%;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav {
+  color: black;
+  margin: 6% 10%;
+}
+
+.container > h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  padding-top: 20%;
+  padding-left: 5%;
+  margin-bottom: unset;
+}
+
+.mobilepara {
+  flex-direction: column;
+}
+
+.mobilepara > p {
+  margin: 5%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.social_media_icons {
+  list-style-type: none;
+
+  /* width: 50%;
+  height: 2em; */
+  padding-left: 5%;
+  margin-top: unset;
+}
+
+.diff_classes > li {
+  list-style-type: none;
+  display: inline;
+  color: #6070ff;
+}
+
+.mobilepara > ul > li {
+  display: inline;
+  height: 2em;
+  width: 50%;
+  padding-left: 5%;
+}
```

Comment:
[OPTIONAL]
**Don't Repeat Yourself**
- Keep your code optimized and reduce same lines of code.

Reply:
This is optional comment. Will follow this in next project